packages/installations/src/helpers/idb-manager.ts

```diff
@@ -33,6 +33,8 @@ const dbPromise: Promise<DB> = openDb(
     switch (upgradeDB.oldVersion) {
       case 0:
         upgradeDB.createObjectStore(OBJECT_STORE_NAME);
+        break;
```

Comment:
Lack of `break` statements here is intentional. Please see the comment above the `switch` statement.

Reply:
oops. will fix.